lib/vmpooler/api.rb

```diff
@@ -168,6 +168,45 @@ def hostname_shorten hostname
           JSON.pretty_generate(result)
         end
 
+        get '/status/?' do
+          content_type :json
+
+          result = {}
+          clone_times = []
+
+          result['capacity_current'] = 0
+          result['capacity_total'] = 0
+
+          result['status'] = 1
+
+          $config[:pools].each do |pool|
+            result['capacity_current'] += $redis.scard( 'vmpooler__ready__' + pool['name'] ).to_i
```

Comment:
If the key happens to be missing, Redis will return a '0' integer and result['capacity_current'] will remain the same value.